handlers/product-move-api/src/main/scala/com/gu/productmove/zuora/GetSubscription.scala

```diff
@@ -25,9 +27,30 @@ trait GetSubscription :
 
 object GetSubscription {
 
-  case class GetSubscriptionResponse(id: String, accountId: String)
+  case class GetSubscriptionResponse(id: String, accountId: String, ratePlans: List[RatePlan])
```

Comment:
probably split this.  Also use 2 space indent on the other case classes in this PR.
```suggestion
  case class GetSubscriptionResponse(
    id: String,
    accountId: String,
    ratePlans: List[RatePlan]
  )
```